### PR TITLE
jv-kongres keyboard

### PR DESCRIPTION
**Pull request by bennylin**
This is a rather simple transliteration keyboard that resulted from the Kongres Aksara Jawa I 2021. Users are required to memorize a lot of keys, and the typing speed is subpar from Nulisa transliteration method (jv-transliteration).

### CODE REVIEW

**Comment by bennylin** [comment]
What an elusive bug! There should be a documentation that these special characters need to be escaped!
$%^&*()_+[]{};':",./<>?

while for some reasons, these do not `~!@#

File: rules/jv/jv-kongres.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			[ '\\&', '꧄' ],
+			[ '\\*', '꧅' ],
+			[ '\\(', '꧌' ],
+			[ '\\)', '꧍' ],
+			[ '\\_', 'ꦷ' ], //wulu melik
+			[ '\\+', 'ꦙ' ], //ja mahaprana
+
+			[ '\\[', 'ꦁ' ], //cecak
+			[ '\\]', 'ꦂ' ], //layar
+			//[ '\', '' ],
```
Comment (labdajiwa) [inline rules/jv/jv-kongres.js:103]: @bennylin The "\\" key for [ZWNJ](https://en.wikipedia.org/wiki/Zero-width_non-joiner) is not to be set? If you are looking for the character, here you go: https://0x0.st/HTwl.txt

Comment (labdajiwa) [reply rules/jv/jv-kongres.js:103]: I'm cross-checking the keymaps and it need one more keymap for ZWNJ (\\). Beside that, they are looks good to me 👍

Comment (bennylin) [reply rules/jv/jv-kongres.js:103]: ok thanks!

Comment (labdajiwa) [reply rules/jv/jv-kongres.js:103]: @bennylin Here is the patch for it.
```suggestion
			[ '\\\\', '‌' ], //zwnj - zero-width non-joiner
```

I just tested it and works as expected. Best regards.

**Review by labdajiwa — APPROVED** [review]

**Review by srish — APPROVED** [review]

**Review by srish — APPROVED** [review]
Approving this patch, but it might be ideal to transfer the documentation from https://bennylin.github.io/kongres/jawa_standar_readme.htm to the wiki and update the URL in the code later.

**Pull request merged**